include/TrigScint/Event/QIEStream.h

```diff
@@ -0,0 +1,143 @@
+#ifndef TRIGSCINT_EVENT_QIESTREAM_H
+#define TRIGSCINT_EVENT_QIESTREAM_H
+
+//---< ROOT >---//
+#include "TObject.h" 
+#include "TrigScint/Event/TrigScintQIEDigis.h"
+
+template <uint32_t N>
+struct mask {
+  static const uint32_t one{1};
+  static const uint32_t m = (one << N) - one;
+};
+template <uint8_t n>
+struct mask8 {
+  static const uint8_t one{1};
+  static const uint8_t m = (one << n) - one;
+};
```

Comment:
A general mask class is available in `packing::utility::mask<N>`, obviously these classes are small, so I don't see a reason for you to remove these if you don't want to. Just putting it on your radar for post-test-beam code cleaning.